### DISCUSSION TITLE
Main folder website address

### DISCUSSION THREAD

**Discussion by robotfx** [FreshRSS/FreshRSS#4344] [Q&A]
Is there any way to have the script load in the main URL address (ie not https://demo.freshrss.org/i/, only https://demo.freshrss.org/). I'm on nginx, if it matters, php 7.3.
Thank you.

**Comment by Alkarex** [comment]
You would have to make a few adjustments to the code itself to do that, or perform some kind of URL rewriting. That is not supported and would end up being a bit less efficient (e.g. caching, cookies, landing experience, ability to perform HTTP Basic authentication...).
Note that there is an automatic temporary redirection to `/i/` (and a waiting landing page on `/` while loading `/i/`) so you are encouraged to use https://demo.freshrss.org/ in your bookmarks and other links.
